### PR TITLE
chore: bump cross-spawn version to be at least 7.0.5

### PR DESCRIPTION
**Pull request by bobbyu99**
#### Description of changes

This PR is to fix [this dependabot alert](https://github.com/aws-amplify/amplify-category-api/security/dependabot/146). In shot, `cross-spawn` is used by `husky` and needs to be bumped to be higher than `7.0.5` for a security patch. 
- bumping `husky` to `^5.0.0` and `cross-spawn` to `7.0.6`. 

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] E2E test run linked
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] Any CDK or CloudFormation parameter changes are called out explicitly

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by tejas2008 — APPROVED** [review]

File: packages/amplify-data-construct/.jsii [hunk 1]
```diff
@@ -3991,7 +3991,7 @@
     "stability": "stable"
   },
   "homepage": "https://github.com/aws-amplify/amplify-category-api.git",
-  "jsiiVersion": "5.5.4 (build 1378d94)",
+  "jsiiVersion": "5.6.0 (build 7be6ace)",
```
Comment (dpilch) [inline packages/amplify-data-construct/.jsii:3994]: Why is the JSII version changing?

Comment (bobbyu99) [reply packages/amplify-data-construct/.jsii:3994]: I merged changes from main, then I ran `yarn build` and `yarn test`. That change appears after running `yarn build`.

Comment (dpilch) [reply packages/amplify-data-construct/.jsii:3994]: It probably shouldn't be changing from modifying the husky version.

Comment (bobbyu99) [reply packages/amplify-data-construct/.jsii:3994]: lemme try again

**Pull request closed without merging**